setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     author="Justin Grilli",
-    author_email="jgrilli@hover.to",
+    author_email="justin.grilli@gmail.to",
```

Comment:
@JustinGrilli  Were you trying to put your gmail.com address?  I've never seen `gmail.to`

Reply:
Fixed!